<launch>
  <node name="gng_planner" pkg="master_project" type="gng_planner.py" output="screen" ns="gng" > 
    <param name="max_nodes" value="2000" /> 
    <param name="robot_radius" value="0.5" /> 
  </node>
</launch>